jmix-flowui/flowui/src/main/java/io/jmix/flowui/xml/layout/loader/component/GenericFilterLoader.java

```diff
@@ -239,4 +251,52 @@ protected ActionLoaderSupport getActionLoaderSupport() {
 
         return actionLoaderSupport;
     }
+
+    protected void applyFilterIfNeeded() {
+        getComponentContext().addInitTask((context, view) -> {
```

Comment:
Filter shouldn't load data if a user don't want to load data, just add conditions